### PR TITLE
Sapper FAQ

### PR DESCRIPTION
**Pull request by antony**
Identikit Sapper FAQ

### Before submitting the PR, please make sure you do the following
- [x] It's really useful if your PR relates to an outstanding issue, so please reference it in your PR, or create an explanatory one for discussion. In many cases features are absent for a reason.
- [x] This message body should clearly illustrate what problems it solves. If there are related issues, remember to reference them.
- [x] Ideally, include a test that fails without this PR but passes with it. PRs will only be merged once they pass CI. (Remember to `npm run lint`!)
### Tests
-  [x] Run the tests tests with `npm test` or `yarn test`)

### Then
- [ ] Remove FAQ item from Wiki and redirect

### CODE REVIEW

File: site/src/utils/slug.js [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+	slugify(string, {
+		customReplacements: [	// runs before any other transformations
+			['$', 'DOLLAR'], // `$destroy` & co
+			['-', 'DASH'], // conflicts with `separator`
+		],
+		separator: SLUG_SEPARATOR,
+		decamelize: false,
+		lowercase: false
+	})
+		.replace(/DOLLAR/g, '$')
```
Comment (benmccann) [inline site/src/utils/slug.js:16]: nit: I might indent this line and the next one less

**Comment by benmccann** [comment]
Maybe we should also have a pointer to the Svelte FAQs on this page? Most of them would be relevant for Sapper users. Before users would automatically see them because both FAQs were on one page

File: site/src/utils/slug.js (hunk 1 above)
Comment (antony) [reply site/src/utils/slug.js:16]: I didn't actually touch that file at all, just copypasta from the Svelte site, so I'm going to leave it as-is :)

**Comment by antony** [comment]
@benmccann and vice-versa perhaps

**Comment by benmccann** [comment]
Doesn't necessarily hurt, but I'm not sure it's as important in the reverse direction. All users of sapper are users of svelte, but not vice versa. But it never hurts to add more documentation and links!

**Comment by benmccann** [comment]
@antony just a quick reminder that it might be useful to add a link to the svelte faq here if you get a chance

File: site/src/routes/faq/index.svelte [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+		<article class='faq'>
+			<h2>
+			<span id={faq.fragment} class="offset-anchor"></span>
+			<a class="anchor" rel='prefetch' href='faq#{faq.fragment}' title='{faq.question}'>&nbsp;</a>
+			{faq.metadata.question}
+			</h2>
+			<p>{@html faq.answer}</p>
+		</article>
+	{/each}
+</div>
```
Comment (antony) [inline site/src/routes/faq/index.svelte:35]:
```suggestion
<p>See also the <a href="https://svelte.dev/faq" rel="external">Svelte FAQ</a> for questions relating to Svelte directly.</p>
</div>
```

**Comment by Conduitry** [comment]
Part of the filename is used in the hash portion of the URL for deeplinking to that question, correct? We should have something more correct here than `100-im-new-to-svelte.md`. I don't have a strong opinion about what this should be.

**Comment by antony** [comment]
> Part of the filename is used in the hash portion of the URL for deeplinking to that question, correct? We should have something more correct here than `100-im-new-to-svelte.md`. I don't have a strong opinion about what this should be.

Oops! Fixed.

**Pull request merged**